src/Libraries/DSCPython/CPythonEvaluator.cs

```diff
@@ -447,6 +447,51 @@ private static string GetTraceBack(Exception e)
 
         #region Marshalling
 
+        /// <summary>
+        /// Add additional data marshalers to handle host data.
+        /// </summary>
+        [SupressImportIntoVM]
+        public override void RegisterHostDataMarshalers(object dataMarshaler)
```

Comment:
can we introduce an interface for this object? Maybe I'm missing something but it seems really hard to write one of these right now without knowing what it should do?

Reply:
I imagine it should go in interfaces in DynamoServices module.

Reply:
So this was moved here from dynamorevit right ?
https://github.com/DynamoDS/DynamoRevit/blob/RC3.3.0_Revit2025/src/DynamoRevit/Models/RevitDynamoModel.cs#L486

Reply:
I wonder why we still need the explicit 'unwrap' function in python ? Couldn't we just optionally apply it to the InputDataMarshaler or OutputDataMarshaler (if that makes sense) ? 
FYI @mjkkirschner @aparajit-pratap

Reply:
also sounds like the marshaling of inputs/outputs should be common for all python engines...
Maybe some of this logic should be moved to the baseclass PythonEngine.

Reply:
> but it seems really hard to write one of these right now without knowing what it should do?
Well, yes. I made it internal. It was kind of stretch to have it public and it's specific to our hosts only.